packages/rocketchat-videobridge/client/tabBar.js

```diff
@@ -0,0 +1,41 @@
+Meteor.startup(function() {
+	Tracker.autorun(function() {
+		if (RocketChat.settings.get('Jitsi_Enabled')) {
+
+			// Load from the jitsi meet instance.
+			if (typeof JitsiMeetExternalAPI !== undefined) {
+				let domain = RocketChat.settings.get('Jitsi_Domain');
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["domain" is defined but never used](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2734190371/issues/source?bid=3199634&fileBranchId=3342079#l7)